compiler/gopherjspkg/doc.go

```diff
@@ -0,0 +1,12 @@
+// Package gopherjspkg provides core GopherJS packages via a virtual filesystem.
+//
+// Core GopherJS packages are packages that are critical for GopherJS compiler
+// operation. They are needed to build the Go standard library with GopherJS.
+// Currently, they include:
+//
+// 	github.com/gopherjs/gopherjs/js
+// 	github.com/gopherjs/gopherjs/nosync
+//
+package gopherjspkg
+
+//go:generate vfsgendev -source="github.com/gopherjs/gopherjs/compiler/gopherjspkg".FS -tag=gopherjsdev
```

Comment:
Where does `vfgendev` come from?

Reply:
See https://github.com/gopherjs/gopherjs/wiki/Developer-Guidelines#code-generation.

Reply:
That's out of scope of this PR. This `gopherjspkg` package is the same in structure as `natives`.